### PR TITLE
[MU3] Fixed linux schedule workflow on CI

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

File: .github/workflows/ci_linux.yml [hunk 1]
```diff
@@ -95,7 +95,7 @@ jobs:
         T_ID=${{ secrets.TELEMETRY_TRACK_ID }}; if [ -z "$T_ID" ]; then T_ID="''"; fi
         YT_API_KEY=${{ secrets.YOUTUBE_API_KEY }}; if [ -z "$YT_API_KEY" ]; then YT_API_KEY="''"; fi
         sudo bash ./build/ci/linux/build.sh -n ${{ github.run_id }} --telemetry $T_ID --youtube_api_key $YT_API_KEY
-      name: Package
+    - name: Package
```
Comment (Jojo-Schmitz) [inline .github/workflows/ci_linux.yml:98]: What is that doing? I had no problems so far without that change

Comment (Jojo-Schmitz) [reply .github/workflows/ci_linux.yml:98]: Or is it because I didn't try to create a pagkage yet? (Well, I tried, but failed, maybe because of this?)

Comment (Jojo-Schmitz) [reply .github/workflows/ci_linux.yml:98]: Oh, that's the master branch, so sorry for the noise

Comment (igorkorsukov) [reply .github/workflows/ci_linux.yml:98]: I notice what Linux nightly build is broken for MU3
https://github.com/musescore/MuseScore/actions/runs/1264459893

I don't know when this happened, but here's a syntax error

Workflow for nightly builds MU3 is located in the master (only from the master, workflows can be started on a schedule). This workflow runs on a schedule and builds the `x.3` branch

Comment (Jojo-Schmitz) [reply .github/workflows/ci_linux.yml:98]: I see, thanks

Comment (Jojo-Schmitz) [reply .github/workflows/ci_linux.yml:98]: And whatever it was, it broke on July 8th, last Linux nightly is from July 7th

Comment (Jojo-Schmitz) [reply .github/workflows/ci_linux.yml:98]: It did for macOS too BTW, so that needs to get fixed too?

Comment (Jojo-Schmitz) [reply .github/workflows/ci_linux.yml:98]: Excact same error in .github/workflows/ci_macos.yml, line 96

Comment (Jojo-Schmitz) [reply .github/workflows/ci_linux.yml:98]: See #9256

**Pull request merged**